src/table/src/requests.rs

```diff
@@ -56,6 +56,7 @@ pub fn validate_table_option(key: &str) -> bool {
         TTL_KEY,
         STORAGE_KEY,
         COMMENT_KEY,
+        AUTO_CREATE_TABLE_KEY,
```

Comment:
It's better to filter out the `auto_create_table` options in `create_non_logical_table`.